### PR TITLE
Ewl-573 livecomments

### PR DESCRIPTION
**Pull request by val3ntin**
a twitter card:
`components/Cards/Twitter`

live comments module:
`modules/LiveComents`

### CODE REVIEW

File: src/components/Card/Twitter.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const tweetId = getTweetIdFromUrl(tweetUrl);
+    if (tweetId === null) return;
+    window.twttr.widgets.createTweet(tweetId.toString(), this.tweetRef.current);
+  };
+
+  render() {
+    const { tweetUrl, className } = this.props;
+
+    return (
+      <div ref={this.tweetRef} key={tweetUrl} className={className}>
```
Comment (val3ntin) [inline src/components/Card/Twitter.js]: need passing in `className` to override TwitterCard styles by styled ( e.g. styled(Twitter)``)

File: src/modules/LiveComments/SharedLink.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import Cards from '../../components/Card';
+
+const SharedLink = ({ sharedlink, labelPlayButton = 'watch' }) => {
+  const sharedLinkProps = {
+    title: get(sharedlink, ['title'], ''),
+    labelPlayButton: get(sharedlink, ['link', 'type'], '') === 2 ? labelPlayButton : '',
+    image: `${get(sharedlink, ['pictureurl'], '')}?w=640`,
```
Comment (davidgarry) [inline src/modules/LiveComments/SharedLink.js]: ?w=640 should be there only if there is a pictureurl

File: src/modules/LiveComments/LiveComment.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    border-radius: 4px;
+  }
+`;
+
+const LiveComment = ({ liveComment, labelPlayButton }) => {
+  const iconUrl = get(liveComment, ['icon', 'svg'], null);
+  const pictureUrl = get(liveComment, ['picture', 'format', 'url'], null);
+  const shouldDisplayComment = !!liveComment.html || !!liveComment.marker || !!iconUrl || !!liveComment.tweet;
+
+  if (!shouldDisplayComment) return null;
```
Comment (davidgarry) [inline src/modules/LiveComments/LiveComment.js]: This test is redundant with the one just below

Comment (val3ntin) [reply src/modules/LiveComments/LiveComment.js]: 🤣 indeed )

File: src/modules/LiveComments/SharedLink.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import Cards from '../../components/Card';
+
+const SharedLink = ({ sharedlink, labelPlayButton = 'watch' }) => {
```
Comment (arturkin) [inline src/modules/LiveComments/SharedLink.js]: Move `labelPlayButton = 'watch' ` to `SharedLink.defaultProps`

Comment (val3ntin) [reply src/modules/LiveComments/SharedLink.js]: removed it at all. as `labelPlayButton` is a required.
thx !

**Pull request merged**